src/lib/rest/rest.cpp

```diff
@@ -711,11 +711,20 @@ int servicePathCheck(ConnectionInfo* ciP, const char* servicePath)
     return 1;
```

Comment:
You should include an entry in CHANGES_NEXT_RELEASE file about the changes in this PR. For instance:

```
- Fix: 400 Bad Request when attempting to DELETE in multi-servicepath cases (instead of silently failing) (#3071)
```

Reply:
Fixed in cdab795